src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -784,97 +785,122 @@ function Test-ApplicationGatewayCRUD3
 	try
 	{
 		# Create the resource group
-		$resourceGroup = New-AzResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "APPGw tag"}
+		$resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "APPGw tag"}
 		# Create the Virtual Network
-		$gwSubnet = New-AzVirtualNetworkSubnetConfig -Name $gwSubnetName -AddressPrefix 10.0.0.0/24
-		$vnet = New-AzvirtualNetwork -Name $vnetName -ResourceGroupName $rgname -Location $location -AddressPrefix 10.0.0.0/16 -Subnet $gwSubnet
-		$vnet = Get-AzvirtualNetwork -Name $vnetName -ResourceGroupName $rgname
-		$gwSubnet = Get-AzVirtualNetworkSubnetConfig -Name $gwSubnetName -VirtualNetwork $vnet
+		$gwSubnet = New-AzureRmVirtualNetworkSubnetConfig -Name $gwSubnetName -AddressPrefix 10.0.0.0/24
+		$vnet = New-AzureRmvirtualNetwork -Name $vnetName -ResourceGroupName $rgname -Location $location -AddressPrefix 10.0.0.0/16 -Subnet $gwSubnet
+		$vnet = Get-AzureRmvirtualNetwork -Name $vnetName -ResourceGroupName $rgname
+		$gwSubnet = Get-AzureRmVirtualNetworkSubnetConfig -Name $gwSubnetName -VirtualNetwork $vnet
+
+		# Create Managed Identity
+		$identity = New-AzureRmUserAssignedIdentity -Name $identityName -Location $location -ResourceGroup $rgname
```

Comment:
@akshaysngupta if you're using the MSI cmdlets in your tests, you'll want to make sure that you add the MSI project to the Network solution so that it gets built along with Network

Reply:
ManagedServiceIdentity proj is already in Network.sln